### DISCUSSION TITLE
How to select not all rows, but only the cell (so NOT 's' and 't')

### DISCUSSION THREAD

**Discussion by emilBeBri** [saulpw/visidata#2016] [General]
So there are a couple of threads here about pasting and selecting cells - Maybe I'm just too thickheaded, but they don't seem to address this very simple use case:

I need to copy the contents not of rows, but of the cells I'm on. when using 's' and 't', they select the entire rows, so that gY copies not only the current columns value but all columns. 

I need to select only the cell I'm in, not the row - this is so simple I can't believe I have to ask about it, but have been looking at the docs and searching and I can't figure out how to do it.  Thank you.

**Comment by saulpw** [comment]
You can't "select" only the cells--VisiData is not a spreadsheet.  But you can select the rows, and then copy the contents of the cells in the current column, with `gzy` or `gzY` (or `zy` or `zY` for only the current cell).

**Comment by emilBeBri** [comment]
Ah right I think I seem to fall back on the spreadsheet metafor alot since mostly I'm using visidata as a replacement for that (data manipulation I tend to do in R), I need to wrap my head around a different way of thinking here.

Thank you!
